lib/bundler/cli.rb

```diff
@@ -317,6 +317,8 @@ def info(gem_name)
       "Binstub destination directory (default bin)"
     method_option "shebang", :type => :string, :banner =>
       "Specify a different shebang executable name than the default (usually 'ruby')"
+    method_option "locked", :type => :boolean, :default => Bundler.bundler_major_version >= 2, :banner =>
```

Comment:
can this use a feature flag instead of directly checking the version?

Reply:
👍 I added a commit to use a feature flag instead of directly checking the version